apps/js/testing_interface.js

```diff
@@ -356,5 +356,48 @@ $(document).ready(function () {
                 errorMsg('Can only paste at a specific location; only select *one* cell as paste destination.');
             }
         }
+        // Crop code
+        if (event.which == 82) {
+            // Press R
```

Comment:
Ok, but this means the feature is not discoverable by a user other than you... the R key should be advertised (in the same place as the C and V commands)